tests/test-project/package.json

```diff
@@ -178,7 +178,8 @@
 		"compile": "tsc -p ./ && npm run lint",
 		"lint": "eslint src --ext .ts",
 		"cb-init": "echo hello_ExTester | clipboard",
-		"ui-test": "npm run cb-init && extest setup-and-run './out/test/cli/order-3.test.js' './out/test/cli/order-2.test.js' './out/test/cli/order-1.test.js' './out/test/**/*.test.js' './out/test/system/clipboard.test.js' -u -i -r . -e ./test-extensions"
+		"ui-test": "npm run cb-init && extest setup-and-run './out/test/cli/order-3.test.js' './out/test/cli/order-2.test.js' './out/test/cli/order-1.test.js' './out/test/**/*.test.js' './out/test/system/clipboard.test.js' -u -i -r . -e ./test-extensions",
+		"ui-coverage": "npm run cb-init && extest setup-and-run './out/test/cli/order-3.test.js' './out/test/cli/order-2.test.js' './out/test/cli/order-1.test.js' './out/test/**/*.test.js' './out/test/system/clipboard.test.js' -u -i -r . -e ./test-extensions --coverage"
```

Comment:
is it possible to not execute also the tests and just only get the coverage?

Reply:
thanks for clarification!

in that case for the npm scripts I think we do not need another script (ui-coverage).. we can just extend `ui-test` with `--coverage` in gh main workflow it would be maybe interesting store the coverage report for one only because its matrix

Reply:
I have removed `ui-coverage` npm script from the test-project's package json. I kept `coverage` script in main package.json, but redefined it to
```
"npm run test -- -- --coverage
```

Reply:
> I have removed `ui-coverage` npm script from the test-project's package json. I kept `coverage` script in main package.json, but redefined it to
> 
> ```
> "npm run test -- -- --coverage
> ```

can you please explain me the purpose of `-- --`? I see that for the first time

Reply:
> > I have removed `ui-coverage` npm script from the test-project's package json. I kept `coverage` script in main package.json, but redefined it to
> > ```
> > "npm run test -- -- --coverage
> > ```
> 
> can you please explain me the purpose of `-- --`? I see that for the first time

This is explained in [this StackOverflow page](https://stackoverflow.com/questions/43046885/what-does-do-when-running-an-npm-command), i.e. `-- --coverage` after the first `--` will be passed to `npm run test`. 

Since `npm run test`  is defined as `npm run ui-test --workspace=extester-test`, 
```
npm run ui-test --workspace=extester-test -- --coverage
```
will be executed. Then `--coverage` is passed to `npm run ui-test --workspace=extester-test` and finally
```
npm run cb-init && extest setup-and-run './out/test/cli/order-3.test.js' './out/test/cli/order-2.test.js' './out/test/cli/order-1.test.js' './out/test/**/*.test.js' './out/test/system/clipboard.test.js' -u -i -r . -e ./test-extensions --coverage
```
will be executed.

Reply:
I never used `-- --` but apparently my guess was right, pass args two levels down instead of just one. Cool.